rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerEventReceiver.java

```diff
@@ -94,7 +99,15 @@ public void start() throws IOException {
     private void openAndReceive(final SchedulerEventListener eventListener, boolean myEventsOnly,
             SchedulerEvent... events) throws IOException {
         Client client = ClientFactory.getDefault().newClient();
-        socket = client.create(client.newOptionsBuilder().reconnect(false).build());
+        AsyncHttpClientConfig.Builder builder = new AsyncHttpClientConfig.Builder();
+        if (insecure) {
+            builder = builder.setAcceptAnyCertificate(true)
+                             .setHostnameVerifier(SSLConnectionSocketFactory.ALLOW_ALL_HOSTNAME_VERIFIER);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "ALLOW_ALL_HOSTNAME_VERIFIER"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)